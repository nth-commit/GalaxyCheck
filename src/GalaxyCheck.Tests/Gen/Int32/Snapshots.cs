using Xunit;
using G = GalaxyCheck.Gen;
using static GalaxyCheck.Tests.TestUtils;

namespace GalaxyCheck.Tests.Gen.Int32
{
    public class Snapshots
    {
        [Fact]
        public void Example() => SnapshotGenValues(G.Int32());

        [Fact]
        public void ExampleOfExampleSpaces() => SnapshotGenExampleSpaces(G.Int32().Between(-10, 10));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant1() => SnapshotGenExampleSpaces(G.Int32().Between(0, 10).ShrinkTowards(5));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant2() => SnapshotGenExampleSpaces(G.Int32().Between(-10, 0).ShrinkTowards(-5));
    }
}
