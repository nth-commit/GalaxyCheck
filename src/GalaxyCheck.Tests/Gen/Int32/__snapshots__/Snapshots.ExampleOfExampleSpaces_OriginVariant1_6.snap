9
..5
..7
....5
....6
......5
..8
....5
....7
......5
......6
........5
